insights/parsers/ndctl_list.py

```diff
@@ -0,0 +1,64 @@
+"""
+NdctlListNi - command ``/usr/bin/ndctl list -Ni``
+=================================================
+"""
+
+from insights.core import JSONParser
+from insights.core.plugins import parser
+from insights.specs import Specs
+
+
+@parser(Specs.ndctl_list)
+class NdctlListNi(JSONParser):
```

Comment:
Since this is a parser for command output, you'd better add the `CommandParser` as the bases.